python/ray/_raylet.pyx

```diff
@@ -635,6 +635,9 @@ cdef execute_task(
             function = execution_info.function
 
             if core_worker.current_actor_is_asyncio():
+                assert len(inspect.getmembers(
+                        actor.__class__,
+                        predicate=inspect.iscoroutinefunction)) > 0
```

Comment:
Simplify this `assert inspect.getmembers(
                        actor.__class__,
                        predicate=inspect.iscoroutinefunction)`